apps/api/src/main/resources/application.yml

```diff
@@ -0,0 +1,11 @@
+spring:
```

Comment:
We use an application.propeties so please chnage that into that format, this will mean in the env you dont have to import like you do below